static/js/redux/constants/colors.ts

```diff
@@ -29,8 +31,7 @@ const COLOUR_DATA = [
   { background: "#E7F76D", highlight: "#C9E20A", border: "#AFC11F", font: "#222" },
   { background: "#A3F5F2", highlight: "#7CD2CF", border: "#53ABA8", font: "#222" },
   { background: "#7499A2", highlight: "#668B94", border: "#6C7A89", font: "#222" },
-  { background: "#E7F76D", highlight: "#C4D44D", border: "#6C7A89", font: "#222" },
```

Comment:
How come this got removed?

Reply:
the background color repeats line 31, so it is very hard to distinguish the slots.

Reply:
oh true, good catch